client/src/app/pages/controls/tags/tags.tsx

```diff
@@ -157,281 +152,283 @@ export const Tags: React.FC = () => {
+          <Table {...tableProps} aria-label="Tag category table">
+            <Thead>
+              <Tr>
+                <TableHeaderContentWithControls {...tableControls}>
+                  <Th {...getThProps({ columnKey: "name" })} />
+                  <Th {...getThProps({ columnKey: "rank" })} />
+                  <Th {...getThProps({ columnKey: "color" })} />
+                  <Th {...getThProps({ columnKey: "tagCount" })} />
+                </TableHeaderContentWithControls>
+              </Tr>
+            </Thead>
+            <ConditionalTableBody
+              isLoading={isFetching}
+              isError={!!fetchError}
+              isNoData={currentPageItems.length === 0}
+              noDataEmptyState={
+                <EmptyState variant="sm">
+                  <EmptyStateIcon icon={CubesIcon} />
+                  <Title headingLevel="h2" size="lg">
+                    {t("composed.noDataStateTitle", {
+                      what: t("terms.tags").toLowerCase(),
+                    })}
+                  </Title>
+                  <EmptyStateBody>
+                    {t("composed.noDataStateBody", {
+                      what: t("terms.tags").toLowerCase(),
+                    })}
+                  </EmptyStateBody>
+                </EmptyState>
+              }
+              numRenderedColumns={numRenderedColumns}
+            >
+              {currentPageItems?.map((tagCategory, rowIndex) => {
+                const hasTags = tagCategory.tags && tagCategory.tags.length > 0;
+                const categoryColor =
+                  tagCategory.colour ||
+                  getTagCategoryFallbackColor(tagCategory);
+
+                return (
+                  <Tbody
+                    key={tagCategory.id}
+                    isExpanded={isCellExpanded(tagCategory)}
                   >
-                    {t("actions.createTagCategory")}
-                  </Button>
-                </ToolbarItem>
-              </RBAC>
-            </ToolbarGroup>
-          }
-          noDataState={
-            <NoDataEmptyState
-              // t('terms.tagCategories')
-              title={t("composed.noDataStateTitle", {
-                what: t("terms.tagCategories").toLowerCase(),
+                    <Tr {...getTrProps({ item: tagCategory })}>
+                      <TableRowContentWithControls
+                        {...tableControls}
+                        item={tagCategory}
+                        rowIndex={rowIndex}
+                      >
+                        <Td width={25} {...getTdProps({ columnKey: "name" })}>
+                          {tagCategory.name}
+                        </Td>
+                        <Td width={10} {...getTdProps({ columnKey: "rank" })}>
+                          {tagCategory.rank}
+                        </Td>
+                        <Td width={10} {...getTdProps({ columnKey: "color" })}>
+                          <Color hex={categoryColor} />
```

Comment:
Something odd with this component such that it doesn't line up with the other text in the row.

Reply:
but also not a regression -- so that can be a followup PR as well

Reply:
https://github.com/konveyor/tackle2-ui/issues/1825